editor/inspector/assets/fbx/model.js

```diff
@@ -695,10 +925,55 @@ exports.update = function(assetList, metaList) {
 };
 
 exports.close = function() {
+    Editor.Message.removeBroadcastListener('fbx-inspector:apply', this.apply.bind(this));
+
     for (const prop in Elements) {
         const element = Elements[prop];
         if (element.close) {
             element.close.call(this);
         }
     }
 };
+
+function getLodItemHTML(lodItems, isBuiltin = false) {
+    let lodItemsStr = '';
+    for (const index in lodItems) {
+        const lodItem = lodItems[index];
+        lodItemsStr += `
+<div class="lod-item">
+    <ui-section cache-expand="fbx-mode-lod-item-${index}">
+        <header slot="header" class="lod-item-header">
+            <div class="left">
+                <span>LOD ${index}</span>
+            </div>
+            <div class="middle" ${ index == 0 || lodItems[0].faceCount == 0 ? 'hidden' : '' }>
+                <span class="face-count">Face count(%)</span>
+                <ui-num-input path="faceCount" min="0" max="100" key="${index}"
+                    value="${Editor.Utils.Math.multi(lodItem.faceCount, 100)}"
+                    ${ isBuiltin ? 'disabled' : '' }>
+                </ui-num-input>
+            </div>
+            <div class="right">
+                <div class="triangles">
+                    <span> ${lodItem.triangleCount} Triangles</span>
+                </div>
+                <div class="operator" ${ isBuiltin ? 'hidden' : '' }>
+                    <ui-icon value="add" key="${index}" path="insertLod" tooltip="insert after this LOD"></ui-icon>
```

Comment:
`tooltip` need to be 1i8n key, below too

Reply:
currently just use the English too